documentation/release-notes/topics/version_4.rst

```diff
@@ -0,0 +1,103 @@
+#########
+Version 4
+#########
+
+*******************
+Hyperview 4.0 (TBD)
+*******************
+This section covers significant changes and bug fixes in Hyperview 4.0.x since version 3.14.x.
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+============================================================
+Enhanced Licensed Feature: Connectivity - Circuit Management
+============================================================
+Circuit management is a new component of the connectivity feature. It adds to the existing work done for connection management and documentation and extends that to give customers the ability to group multiple connections into an end-to-end circuit. Some of the capabilities in this feature include:
+
+- Set and manage different circuit types and statuses
+- Extend circuit properties with custom properties
+- Manage access control on different circuits
+- manage sort and search associated connection segments and set A and Z termination points
+- Bulk import data
+
+.. note:: Please contact the Hyperview Sales or Support teams for more information.
+
+=========================================
+Enhanced Licensed Feature: Outlet Control
+=========================================
+Outlet control has been enhanced to allow administrators to control multiple outlets at the same time.
+
+The rack PDU layout has been enhanced to allow the selection of multiple outlets at once and then initiate an action to control turn on, off or cycle selected outlets.
+
+For power-consuming devices, such as servers, the Information -> Power page has been improved to allow for outlet control action on multiple power sources. The page was further enhanced to display the latest available output total power and load for connected Rack PDUs.
```

Comment:
I think it should be "...outlet control actions..." instead of "...outlet control action..."

Reply:
Updated, thanks!